lib/@uncharted.software/stories-facets/src/main.js

```diff
@@ -128,10 +129,11 @@ Facets.prototype.deselect = function(simpleGroups) {
  * @method replace
  * @param {Object} groups - An object describing the groups of facets to be created.
  * @param {Object=} queries - Optional object describing the queries that should be created along with the facets.
+ * @param {boolean=} noTransition - If truthful will disable CSS transitions during init.
```

Comment:
I think you mean "truthy" (the JavaScript term) rather than "truthful" (the English term); also, put a comma after: If truthy, will disable CSS transitions during init.